source/PlayServicesResolver/src/JavaUtilities.cs

```diff
@@ -185,19 +186,12 @@ private static string FindJavaTool(string javaTool)
             float majorMinorVersion = 0;
             // The version string is can be reported via stderr or stdout so scrape the
             // concatenated message string.
-            foreach (var line in CommandLine.SplitLines(result.message)) {
-                if (line.StartsWith("java version ")) {
-                    var tokens = line.Split();
-                    var versionString = tokens[tokens.Length - 1].Trim(new [] { '"' });
-                    var components = versionString.Split(new [] { '.' });
-                    if (components.Length < 2) {
-                        continue;
-                    }
-                    if (!float.TryParse(components[0] + "." + components[1], NumberStyles.Any,
-                                        CultureInfo.InvariantCulture, out majorMinorVersion)) {
-                        continue;
-                    }
-                }
+            string pattern = "^(?<model>java||openjdk) version \"(?<first>\\d).(?<major>\\d).(?<minor>\\d).*$";
```

Comment:
nit: I believe version is semver.  i.e major.minor.revision  Worth renaming the fields?

Reply:
I wasn't sure, fixed it.